mmv1/products/dataplex/Datascan.yaml

```
@@ -68,27 +68,36 @@ examples:
       'fmt.Sprintf("tf-test-datascan%s", context["random_suffix"])'
```

`primary_resource_name` needs to match the name of the resource generated for tests. This also only actually needs to be set on the first example. It's used for IAM tests. Related: https://github.com/hashicorp/terraform-provider-google/issues/8712

```suggestion
      'fmt.Sprintf("tf-test-datascan%s", context["random_suffix"])'
```